src/requestManager.cpp

```diff
@@ -522,39 +539,61 @@ void CRequestManager::SendRequests()
 		    {
                     if (1)
                       {
-                      cs_objDownloader.unlock();
-                      LOCK(next.node->cs_vSend);
-  		      // from->AskFor(item.obj); basically just shoves the req into mapAskFor
-                      if (!item.lastRequestTime || (item.lastRequestTime && IsChainNearlySyncd()))
-                        {
-                          next.node->mapAskFor.insert(std::make_pair(now, item.obj));
-                          item.outstandingReqs++;
-		          item.lastRequestTime = now;
-                        }
-                      cs_objDownloader.lock();
-		      }
-                    if (0)  // SAME NODE RETRY DISABLED: Won't help in a TCP connection anyway: 
```

Comment:
WRT removal of "if (0)" and subsequent logic: I want to preserve my thought process here and keep the code logically correct in case we move to multiple transports...